pkg/client/interfaces.go

```diff
@@ -91,4 +91,6 @@ type Interface interface {
 	GetStatus(namespace string, client kubernetes.Interface) (*aggregation.Status, error)
 	// GetLogs streams logs from the sonobuoy pod by default to stdout.
 	GetLogs(cfg *LogConfig, client kubernetes.Interface) error
+	// GetTests returns a collection of tests extracted from a sonobuoy archive.
+	GetTests(reader io.Reader, show string) (TestCases, error)
```

Comment:
idk how i feel about this being part of the interface. Seems like we were on the fence including it in the CLI anyway, does it make sense to be part of the SB interface or is this a special case?

Reply:
Put a //TODO around it b/c long term it should be it's own thing.